hazelcast/src/main/java/com/hazelcast/transaction/impl/xa/XATransactionImpl.java

```diff
@@ -78,8 +78,8 @@
     private final SerializableXID xid;
     private final String txOwnerUuid;
 
-    private final List<TransactionRecord> txLogs = new LinkedList<TransactionRecord>();
-    private final Map<Object, TransactionRecord> txLogMap = new HashMap<Object, TransactionRecord>();
+    private final List<TransactionRecord> records = new LinkedList<TransactionRecord>();
```

Comment:
the name logs is wrong since it stores records. The transaction is all about transacrtion records and if you prefix it all the time with transaction records, the code starts to become very verbose for no real reason.